app/models/concerns/foreman_remote_execution/exportable.rb

```diff
@@ -12,37 +12,42 @@ module ForemanRemoteExecution
   module Exportable
     extend ActiveSupport::Concern
 
-    def to_export
+    def to_export(include_blank = true)
       self.class.exportable_attributes.keys.inject({}) do |hash, attribute|
-        value = export_attr(attribute, self.class.exportable_attributes[attribute])
+        value = export_attr(attribute, self.class.exportable_attributes[attribute], include_blank)
 
-        # Rails considers false blank, but if a boolean value is explicitly set false, we want to ensure we export it.
-        (value.blank? && value != false) ? hash : hash.update(attribute => value)
+        if include_blank
```

Comment:
yeah you could either wrap it in #tap block or just return hash just after the conditional update... anyway whatever works for you... it would be great if you could update the same in Foreman PR, anyway that does not block this PR